<launch>

  <node pkg="joy" name="joy_node" type="joy_node">
    <remap from="joy" to="joy/raw"/>
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="autorepeat_rate" type="double" value="120"/>
    <param name="deadzone" type="double" value="0"/>
  </node>

  <node pkg="operator_node" name="joy_remap_node" type="joy_remap_node">
    <remap from="joy_in" to="joy/raw"/>
    <remap from="joy_out" to="joy"/>
    <param name="hz" type="int" value="100"/>
  </node>

  <node pkg="operator_node" name="operator_node" type="isometric_node" output="screen">
    <param name="max_velocity" type="double" value="0.04"/>
    <rosparam param="axis">[0, 1]</rosparam>
    <rosparam param="directions">[1, -1]</rosparam>    
  </node>

  <node pkg="operator_node" name="logger" type="interface_logger_node" output="screen">
    <param name="window_duration" type="double" value="1.0"/>
  </node>  

</launch>
